src/assets/icon/IconCheck.js

```diff
@@ -1,17 +1,37 @@
 import * as React from 'react';
-import Svg, {Path, Rect} from 'react-native-svg';
-import {COLORS} from '../../utils/theme';
+import PropTypes from 'prop-types';
+import Svg, {G, Path, Defs, ClipPath, Rect} from 'react-native-svg';
 
-const IconCheck = (props) => (
-  <Svg width={32} height={32} fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
-    <Rect width={32} height={32} rx={16} fill={COLORS.signed_primary} />
-    <Path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M23.699 10.285a1 1 0 0 1 .016 1.414l-9.765 10a1 1 0 0 1-1.479-.053l-4.234-5a1 1 0 0 1 1.526-1.292l3.524 4.16 8.998-9.213a1 1 0 0 1 1.414-.016Z"
-      fill={COLORS.white}
-    />
-  </Svg>
-);
+function IconCheck(props) {
+  return (
+    <Svg
+      viewBox="0 0 24 24"
+      width={props?.width ?? 20}
+      height={props?.height ?? 20}
+      fill={props?.fill}
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}>
+      <G clipPath="url(#clip0_19924_30913)">
+        <Path fill="#4782D7" d="M-4.5 -5.25H28.5V28.5H-4.5z" />
+        <Path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M17.774 7.713a.75.75 0 01.013 1.061l-7.325 7.5a.75.75 0 01-1.109-.04l-3.175-3.75a.75.75 0 111.144-.969l2.643 3.121 6.748-6.91a.75.75 0 011.061-.013z"
+          fill="#fff"
+        />
+      </G>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of grouping (`<G>`) and clipping (`clipPath`) elements can help with SVG manipulation and styling. However, ensure that the `clipPath` id (`clip0_19924_30913`) is unique across the application to avoid potential conflicts.



<!-- This is an auto-generated comment by OSS CodeRabbit -->